src/passes/Memory64Lowering.cpp

```diff
@@ -29,10 +29,6 @@ namespace wasm {
 
 struct Memory64Lowering : public WalkerPass<PostWalker<Memory64Lowering>> {
 
-  void run(PassRunner* runner, Module* module) override {
```

Comment:
Are you disabling this pass for now?

Reply:
Oh, this is just a no-op change. The function was just calling the super, and without this code the super is just called normally.

(If the pass were disabled then the relevant test would break - I verified we have such a test now, `test/passes/memory64-lowering_enable-memory64_enable-bulk-memory_enable-threads.wast`)